### PR TITLE
Add badges to README

### LINKED ISSUE #10: Add badges to README

### PR DESCRIPTION
**Pull request by tobiashuste**
Closes #10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
 -->
 
 # Redis Ansible Role
 
+[![CI Status](https://github.com/hifis-net/ansible-role-redis/actions/workflows/ci.yml/badge.svg)](https://github.com/hifis-net/ansible-role-redis/actions/workflows/ci.yml)
+[![Ansible Galaxy Role](https://img.shields.io/ansible/role/54679?color=orange)](https://galaxy.ansible.com/hifis/redis)
+[![Ansible Galaxy Role downloads](https://img.shields.io/ansible/role/d/54679)](https://galaxy.ansible.com/hifis/redis)
+[![Ansible Galaxy quality score](https://img.shields.io/ansible/quality/54679)](https://galaxy.ansible.com/hifis/redis)
+[![Apache-2.0 Licensed](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/hifis-net/ansible-role-redis/blob/main/LICENSES/Apache-2.0.txt)
+[![Latest release](https://img.shields.io/github/v/release/hifis-net/ansible-role-ssh-keys)](https://github.com/hifis-net/ansible-role-redis/releases)
```
Comment (christianhueserhzdr) [inline README.md]: Dear Tobias @tobiashuste, this link to ssh-keys should be replaced by one directing towards redis:

```suggestion
[![Latest release](https://img.shields.io/github/v/release/hifis-net/ansible-role-redis)](https://github.com/hifis-net/ansible-role-redis/releases)
```
Or shall we refer to the latest tag and not the latest release?

```suggestion
[![Latest tag](https://img.shields.io/github/v/tag/hifis-net/ansible-role-redis)](https://github.com/hifis-net/ansible-role-redis/tags)
```

**Review by christianhueserhzdr — CHANGES_REQUESTED** [review]
Dear Tobias @tobiashuste, a wrong URL has been given regarding the latest release. It points to ssh-keys. Shall we also link the tags here instead of the releases?

File: README.md (hunk 1 above)
Comment (tobiashuste) [reply README.md]: Thanks for noticing :+1: This was already fixed in the version before.

I would stick to releases. As soon as they're created the badge looks fine.

**Review by christianhueserhzdr — APPROVED** [review]
Dear Tobias @tobiashuste, thanks for fixing the URL. I approve this PR.

**Pull request merged**